OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -217,7 +217,7 @@ public void UpdateMouseover()
 
 			if (resourceLayer != null)
 			{
-				var resource = resourceLayer.GetRenderedResource(cell);
+				var resource = resourceLayer.GetResourceType(cell);
```

Comment:
This needs to query `ResourceRenderer` instead of `ResourceLayer` to maintain the previous behaviour.

Reply:
`ResourceLayer` doesn't contain the `ResourceType` anymore.

Reply:
This line is setting up the tooltip that is shown to the player. It needs to match up with what is actually rendered!

Reply:
Updated here and in Harvester.CanTarget.